src/support_sphere_db/src/ssdb/models/abstract_public_schema_model.py

```diff
@@ -0,0 +1,5 @@
+from sqlmodel import SQLModel
+
+
+class AbstractPublicSchemaModel(SQLModel):
```

Comment:
I wouldn't call this an "Abstract" model in a proper sense since it didn't inherit from an abstract base class. I would just say, `BasePublicSchemaModel`